packages/next/build/swc/src/transform.rs

```diff
@@ -117,7 +117,27 @@ where
 {
     let c = get_compiler(&cx);
 
-    let src = cx.get::<JsString>(0)?.into_utf8()?.as_str()?.to_owned();
+    let unknown_src = cx.get::<JsUnknown>(0)?;
+    let src = match unknown_src.get_type()? {
+        napi::ValueType::String => napi::Result::Ok(
+            unsafe { unknown_src.cast::<JsString>() }
+                .into_utf8()?
+                .as_str()?
+                .to_owned(),
+        ),
+        napi::ValueType::Object => napi::Result::Ok(
```

Comment:
This is undefined behavior. 
https://docs.rs/napi/1.7.7/napi/struct.JsUnknown.html#method.cast

Says
>  V must be match with the return value of get_type

but the returned value is `Object`.

Reply:
Also, `unsafe` is not required.


According to the doc, `JsBuffer` implements `TryFrom<JsUnknown>`.
https://docs.rs/napi/1.7.7/napi/struct.JsBuffer.html#impl-TryFrom%3CJsUnknown%3E

```rust
JsBuffer::try_from(unknown_src)?
```

will work.